### PR TITLE
Please review  the code, some wild things are  going on  :-(

### PR DESCRIPTION
**Pull request by Aaliyah379**

### CODE REVIEW

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 1]
```diff
@@ -90,7 +90,7 @@ await _unitOfWork.ExerciseRepository
 
             await _unitOfWork.Save();
 
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction("WorkoutDay","Details",exercise);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: https://stackoverflow.com/questions/10785245/redirect-to-action-in-another-controller

Here is signature of the method
RedirectToRouteResult RedirectToAction (string actionName, string controllerName, object routeValues);

Your should use
RedirectToAction(nameof(ExcerciseController.Details),"WorkoutDays", exercise.WorkoutDayId);

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 2]
```diff
@@ -147,7 +147,7 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,Name,RestTime,WorkoutDay
 
                 throw;
             }
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction("Workoutdays","Details",id);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: the same as in a comment on top

File: MyWorkout.Web/Controllers/ExercisesController.cs [hunk 3]
```diff
@@ -177,7 +177,7 @@ public async Task<IActionResult> DeleteConfirmed(int id)
             await _unitOfWork.ExerciseRepository.Delete(id);
             await _unitOfWork.Save();
 
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction("WorkoutDays","Details",id);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/ExercisesController.cs]: the same as in a comment on top

File: MyWorkout.Web/Controllers/RepeatsController.cs [hunk 4]
```diff
@@ -89,7 +89,7 @@ public async Task<IActionResult> Create([Bind("Id,Number,Count,ExerciseId")] Rep
                 await _unitOfWork.RepeatRepository.Create(repeat);
                 await _unitOfWork.Save();
 
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction("Exercises","Details",repeat);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/RepeatsController.cs]: return RedirectToAction(nameof(ExcersiceController.Details),"Exercises",repeat.ExcerciseId);

File: MyWorkout.Web/Controllers/RepeatsController.cs [hunk 5]
```diff
@@ -145,7 +145,7 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,Number,Count,ExerciseId"
 
                     throw;
                 }
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction("Exercises","Details",repeat);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/RepeatsController.cs]: the same as in a comment on top

File: MyWorkout.Web/Controllers/RepeatsController.cs [hunk 6]
```diff
@@ -180,7 +180,7 @@ public async Task<IActionResult> DeleteConfirmed(int id)
             await _unitOfWork.RepeatRepository.Delete(id);
             await _unitOfWork.Save();
 
-            return RedirectToAction(nameof(Index));
+            return RedirectToAction("Exercises","Details",id);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/RepeatsController.cs]: the same as in a comment on top

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs [hunk 7]
```diff
@@ -85,7 +85,7 @@ public async Task<IActionResult> Create([Bind("Id,DayOfWeek,MuscleGroup,PlanId")
                 await _unitOfWork.WorkoutDayRepository.Create(workoutDay);
                 await _unitOfWork.Save();
 
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction("Plans","Details",workoutDay);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/WorkoutDaysController.cs]: return RedirectToAction(nameof(PlansController.Details),"Plans",workoutDay.PlanId);

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs [hunk 8]
```diff
@@ -136,7 +136,7 @@ public async Task<IActionResult> Edit(int id, [Bind("Id,DayOfWeek,MuscleGroup, P
 
                     throw;
                 }
-                return RedirectToAction(nameof(Index));
+                return RedirectToAction("Plans","Details",workoutDay.Id);
```
Comment (YuriyMorozyuk95) [inline MyWorkout.Web/Controllers/WorkoutDaysController.cs]: the same as in a comment on top

**Review by YuriyMorozyuk95 — CHANGES_REQUESTED** [review]

File: MyWorkout.Web/Controllers/WorkoutDaysController.cs (hunk 8 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/WorkoutDaysController.cs]: Updated

File: MyWorkout.Web/Controllers/RepeatsController.cs (hunk 5 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/RepeatsController.cs]: Updated accordingly

File: MyWorkout.Web/Controllers/ExercisesController.cs (hunk 3 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/ExercisesController.cs]: Updated accordingly

File: MyWorkout.Web/Controllers/RepeatsController.cs (hunk 4 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/RepeatsController.cs]: Updated accordingly

File: MyWorkout.Web/Controllers/RepeatsController.cs (hunk 6 above)
Comment (Aaliyah379) [reply MyWorkout.Web/Controllers/RepeatsController.cs]: updated accordingly

**Pull request merged**